python/lsst/afw/math/boundedField.cc

```diff
@@ -69,11 +69,14 @@ PYBIND11_PLUGIN(_boundedField) {
     clsBoundedField.def("__truediv__", &BoundedField::operator/);
 
     /* Members */
-    clsBoundedField.def("evaluate", (double (BoundedField::*)(double, double) const)
-        &BoundedField::evaluate);
-    clsBoundedField.def("evaluate", (ndarray::Array<double,1,1>
-        (BoundedField::*)(ndarray::Array<double const,1> const &,
-                          ndarray::Array<double const,1> const &) const) &BoundedField::evaluate);
+    clsBoundedField.def("evaluate", (double (BoundedField::*)
+        (double, double) const) &BoundedField::evaluate);
+    clsBoundedField.def("evaluate", (ndarray::Array<double,1,1> (BoundedField::*)
+        (ndarray::Array<double const,1> const &,
+         ndarray::Array<double const,1> const &) const) &BoundedField::evaluate);
+    clsBoundedField.def("evaluate", (double (BoundedField::*)
+        (lsst::afw::geom::Point2D const &) const) &BoundedField::evaluate);
```

Comment:
If you are going to reformat existing code please be sure to use `clang-format` (as per a recent RFC that specifies "google" format and a very few config overrides). That's our new standard. In this case I suspect the old code was already formatted that way. Technically you ought to run that on all C++ code you write (I was tempted to mention that earlier but worried it would be too picky. Oh well...).